content/millennium/r4/scheduling/appointment.md

```diff
@@ -161,6 +161,7 @@ _Implementation Notes_
 * Only operations on the paths listed below are supported.
 * For video visit link patch operation paths, `contained` index 0 represents the provider link and `contained` index 1 represents the patient link.
 * Video Visit linking requires additional client configuration and application support.
+* Additional Video Visit cloud configuration is required to access these patch operations.
```

Comment:
Krystal wanted to have a separation between cloud configuration and client configuration, since they could apply to different things. I can go with 

`Video Visit link patching requires additional client configuration, cloud configuration, and application support.` 

if that sounds better.

Reply:
modified https://github.com/cerner/fhir.cerner.com/pull/385/commits/ab0f399f4c56a6788c04ccd244ad0a9e9453ede9#diff-1d7df8ceb9f0ae8471323516e037c662R163